### DISCUSSION TITLE
第5节课笔记(21班)

### DISCUSSION THREAD

**Discussion by mattheliu** [InternLM/Tutorial#225] [笔记]
请提交笔记链接 Please submit the link to the notes.

**Comment by JinProton** [comment]
https://zhuanlan.zhihu.com/p/679241631

**Comment by jayjayhust** [comment]
https://github.com/jayjayhust/InternLM-Tutorial/blob/main/Lesson5.md

**Comment by water2silver** [comment]
课程笔记与课程作业
https://zhuanlan.zhihu.com/p/679757890

**Comment by cqu-student** [comment]
https://zhuanlan.zhihu.com/p/679754067

**Comment by TonyTnT** [comment]
https://zhuanlan.zhihu.com/p/679783071

**Comment by yunhuisong2023** [comment]
https://reflective-value-ac9.notion.site/LMDeploy-3773d01f0d14485cb2dff660c851834d

**Comment by YongXie66** [comment]
笔记与作业
https://github.com/YongXie66/InternLM_tutorial_copy/blob/main/notes/Lesson5.pdf

**Comment by mattheliu** [comment]
https://github.com/mattheliu/gitblog/issues/5
